openBCIBoard.js

```diff
@@ -172,10 +207,9 @@ function OpenBCIFactory() {
     OpenBCIBoard.prototype.disconnect = function() {
         var timeout = 0;
         if (this.streaming) {
+            this.streamStop();
             if(this.options.verbose) console.log('stop streaming');
-            this.streaming = false;
-            this.write(k.OBCIStreamStop);
-            timeout = 10;
+            timeout = 60;
```

Comment:
The timeout length is really dependent on circumstances, such as if we are streaming, and further, what we are trying to send to the board and the consequences of NOT waiting the full time.